src/main/frontend/extensions/zotero/extractor.cljs

```diff
@@ -152,27 +157,30 @@
 
 (defmethod extract "attachment"
   [item]
-  (let [{:keys [title url link-mode path content-type filename]} (-> item :data)]
+  (let [{:keys [title url link-mode path content-type filename]} (-> item :data)
+        link-func (if (= :org (state/get-preferred-format))
+                    org-link
+                    markdown-link)]
     (case link-mode
       "imported_file"
       (str
-       (markdown-link title (local-link item))
+       (apply link-func title (local-link item))
```

Comment:
the `apply` usage is something like `(apply f args)`, we can remove the `apply` usage here because `link-func` takes a label and a link as it's arguments.